### PR TITLE
fix path and crlf errors on windows

### LINKED ISSUE #433: Various path-related errors on Windows
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

Things break on Windows because nobody bothered to test for it.

### To reproduce

`cargo test` 

### Expected behavior

it works :)

### Environment

Windows lol

### PR DESCRIPTION
**Pull request by nebulatgs**
- fix path and crlf errors on windows
- improve docker test performance

resolves #433

### CODE REVIEW

**Review by neverabsolute — APPROVED** [review]
lgtm

**Review by cubedhuang — APPROVED** [review]
LGTM! :+1:

**Comment by neverabsolute** [comment]
> LGTM! 👍

you're a fan

**Comment by cubedhuang** [comment]
> > LGTM! 👍
> 
> you're a fan

and you're?

**Comment by neverabsolute** [comment]
> > > LGTM! 👍
> > 
> > 
> > you're a fan
> 
> and you're?
IMG_0787

File: tests/docker_run_tests.rs [hunk 1]
```diff
@@ -324,112 +323,105 @@ fn test_node_nx_express() {
...
 
 #[test]
 fn test_python() {
     let name = simple_build("./examples/python");
-    let output = run_image(name, None);
-    assert!(output.contains("Hello from Python"));
-}
-
-#[test]
-fn test_python_procfile() {
```
Comment (nebulatgs) [inline tests/docker_run_tests.rs:423]: why did this test vanish

**Review by JakeCooper — APPROVED** [review]

**Pull request merged**